# please-help-with-this-error
TypeError                                 Traceback (most recent call last) &lt;ipython-input-12-9e53843b0e27> in &lt;module> ----> 1 history = Model.fit_generator(train_generator,       2                               epochs=10,       3                               validation_data=validation_generator,       4                               callbacks=[checkpoint])  ~\anaconda3\envs\tensorflow\lib\site-packages\tensorflow\python\util\deprecation.py in new_func(*args, **kwargs)     322               'in a future version' if date is None else ('after %s' % date),     323               instructions) --> 324       return func(*args, **kwargs)     325     return tf_decorator.make_decorator(     326         func, new_func, 'deprecated',  TypeError: fit_generator() missing 1 required positional argument: 'generator'
